bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/EarlyStartupRegistration.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ *
```

Comment:
Can you explain underlying reason why we need to do this?

Reply:
The need to register the resource change listener on bundle start is to cover the case where someone imports a new project for the first time (no Liberty nature in it already), and, before doing any other Liberty Tools operation (eg using Run Config or dashboard), tries to right-click Run As from the project e.g. in Package Explorer.  

Without the listener active we won't add the nature, and the user won't get the "Run As -> Liberty" sub-menu options.

If there's a better way without an empty class , that'd be fine with me.